README.md

```diff
@@ -141,12 +141,22 @@ sudo apt-get install \
 
 In the folly directory, run:
 ```
-  mkdir _build && cd _build
+  mkdir _build
   cmake ..
   make -j $(nproc)
   make install
 ```
 
+If you receive an error like:
+
+> CMake Error at cmake_install.cmake:41 (file):
+> file INSTALL cannot copy file
+> "/home/[yourname]/[folly-version]/folly/libfolly.a" to
+> "/usr/local/lib/libfolly.a".
+
+then you [need to use `sudo make install`](https://askubuntu.com/a/954877)
```

Comment:
This link does not need to exist - this random answer to a random question is not any kind of canonical or authoritative source text.

Reply:
Okay I will remove it.